src/amo/sagas/userAbuseReports.js

```diff
@@ -0,0 +1,49 @@
+// Disabled because of
```

Comment:
After https://github.com/mozilla/addons-frontend/issues/4323 , I feel like we should add flow types for sagas since they are the "glue" between actions and API calls. It's hard to get these integrations right and our tests can't cover them (we can't make real API requests). This is where type safety really shines.

Reply:
I recall flow annotations on sagas being pretty unintuitive/unhelpful to look at directly so I think that's why I opted out of writing them traditionally. I think it was mostly around the generator functions though.

I'd be fine with a code quality issue for adding flow types to sagas though.